.github/workflows/examples.yml

```
@@ -117,3 +117,89 @@ jobs:
           fi
 
           killall python
+
+  test_pytorch_from_centralized_to_federated:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7.9
+      - name: Install
+        run: |
+          python -m pip install poetry==1.1.4
+          poetry config virtualenvs.create false
+      - name: Install dependencies
+        run: |
+          cd examples/pytorch_from_centralized_to_federated
+          python -m poetry install
+      - name: Cache Datasets
+        uses: actions/cache@v2
+        with:
+          path: './examples/pytorch_from_centralized_to_federated/dataset'
+          key: pytorch-datasets
+      - name: Download Datasets
+        run: |
+          cd examples/pytorch_from_centralized_to_federated
+          python -c "from torchvision.datasets import CIFAR10; CIFAR10(\"./dataset\", download=True)"
+      - name: Import flwr
+        run: |
+          python3 -c "import flwr"
+      - name: Test
+        run: |
+          cd examples/pytorch_from_centralized_to_federated
+
+          python3 server.py &
+          sleep 3
+
+          if [[ $(ps aux | grep "python server.py" | awk '{ print $2 }') ]];
+            then echo "Server process started correctly";
+            else echo "Server process crashed" && exit 1;
+          fi
+
+          python3 client.py 1 &
+          sleep 3
+
+          if [[ $(ps aux | grep "python client.py 1" | awk '{ print $2 }') ]];
+            then echo "Client process 1 started correctly";
+            else echo "Client process 1 crashed" && exit 1;
+          fi
+
+          python3 client.py 2 &
+          sleep 3
+
+          if [[ $(ps aux | grep "python client.py 2" | awk '{ print $2 }') ]];
+            then echo "Client process 2 started correctly";
+            else echo "Client process 2 crashed" && exit 1;
+          fi
+  test_simulation:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7.9
+      - name: Install
+        run: |
+          python -m pip install poetry==1.1.4
+          poetry config virtualenvs.create false
+      - name: Install dependencies
+        run: |
+          cd examples/simulation
+          python -m poetry install
+      - name: Import flwr
+        run: |
+          python3 -c "import flwr"
+      - name: Test
+        run: |
+          cd examples/simulation
+
+          python3 simulation.py &
+          sleep 3
```

```suggestion
          sleep 10
```
Let's give this slightly longer time as it will start the server and various clients.